pkg/controller/scyllacluster/resource.go

```diff
@@ -643,6 +699,52 @@ func StatefulSetForRack(r scyllav1.RackSpec, c *scyllav1.ScyllaCluster, existing
 								},
 							},
 						},
+						{
+							Name:            naming.InterNodeTrafficProbeSidecarContainerName,
+							Image:           sidecarImage,
```

Comment:
It's a different issue, I'm not solving it here. This image is not present in ScyllaCluster so I used what is already there to prevent from pointless image downloading.

Reply:
we can't keep making this worse to fix, please use a different image

Reply:
> we can't keep making this worse to fix,

How does it make it worse? It's not affected by image using rolling tag skew. 

I don't see any benefit of using suggested image. 
It would be hardcoded in binary, making it impossible to change without new release. Which means it may get eventually outdated getting full of CVEs. Image nor repo is controlled by us which implies it may be removed tomorrow. It may not be available in case when non-public restricted repository is used. 

Image used in PR is configurable in runtime. It doesn't add non-existing dependency on bash because we already depend on it in NodeConfig.
Current version of operator image have bash. PR changing it, would have to make sure every usage is aligned. This one isn't very hard to change.